package.json

```diff
@@ -13,7 +13,7 @@
   },
   "peerDependencies": {
     "babel-core": "^6.0.0",
-    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta"
+    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta || 2.1.0-beta.0"
```

Comment:
`^1 || ^2.1.0-beta.2` == `1 || ^2.1.0-beta.2`

That's enough SemVer for me today >.>